source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -17,20 +17,82 @@ To view the end-user facing docs, see the :doc:`feature_release_notes`.
 Breaking Changes
```

Comment:
On Lines 5-6 there are dates for the release span (May 10 - October 23) - are those dates the accurate ones? @cmltaWt0 or @farhaanbukhsh ?

Reply:
It looks like Sumac was cut on November 5th, and Redwood on May 10th. Unfortunately I can't make a suggestion here because the diff is too far up, but L6 should say

"to November 5, 2024"

@feoh

Reply:
My comment is a bit late, but the Sumac master branch was created on October 24.

@sarina ^

Reply:
Redwood master was cut on May 9, 2024.
So the correct line could be `spanning changes from May 9, 2024
to October 24, 2024`.